mrjob/aws.py

```diff
@@ -181,6 +181,23 @@
 _DEFAULT_AWS_REGION = 'us-east-1'
 
 
+
+# if AWS throttles us, how long to wait (in seconds) before trying again?
+_AWS_BACKOFF = 20
+_AWS_BACKOFF_MULTIPLIER = 1.5
```

Comment:
I'm not a fan of limitless backoff. Can we have a max backoff seconds that we respect.

Reply:
It isn't limitless, it will stop after `_AWS_MAX_TRIES` attempts (20)

Reply:
Right but it exponentially backs off, which means that by the 10th retry we are sleeping 

1153 seconds which is like 20 mins. I am saying we should have a max_wait that is something reasonable like 10s.